jtrust-lib/src/main/java/be/fedict/trust/crl/OnlineCrlRepository.java

```diff
@@ -105,7 +106,10 @@ private X509CRL getCrl(final URI crlUri) throws IOException,
 
 		if (null != this.networkConfig) {
 			final HttpHost proxy = new HttpHost(this.networkConfig.getProxyHost(), this.networkConfig.getProxyPort());
+			int timeout = 5; // seconds
```

Comment:
this should be a static final field, and you should probably just specify it in milliseconds